sql-statements/sql-statement-explain.md

```diff
@@ -6,7 +6,8 @@ aliases: ['/docs/dev/sql-statements/sql-statement-explain/','/docs/dev/reference
 
 # `EXPLAIN`
 
-The `EXPLAIN` statement shows the execution plan for a query without executing it. It is complimented by `EXPLAIN ANALYZE` which will execute the query. If the output of `EXPLAIN` does not match the expected result, consider executing `ANALYZE TABLE` on each table in the query.
+The `EXPLAIN` statement shows the execution plan for a query without executing it. It complements the `EXPLAIN ANALYZE` statement, which executes the query. 
+If the output of `EXPLAIN` does not match the expected result, consider executing `ANALYZE TABLE` on each table in the query.
```

Comment:
```suggestion
If the output of `EXPLAIN` does not match the expected result, consider executing `ANALYZE TABLE` on each table in the query to make sure the table statistics are up to date.
```

There can be multiple reasons for mismatches between the `EXPLAIN` and `EXPLAIN ANALYZE` output. However this would still be a good first step.